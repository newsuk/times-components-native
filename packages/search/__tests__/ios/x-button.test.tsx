import React from "react";
import TestRenderer from "react-test-renderer";
import XButton from "../../src/search-bar/x-button";

describe("XButton", () => {
  it("Should render correctly", () => {
    const testInstance = TestRenderer.create(<XButton onPress={() => null} />);
    expect(testInstance).toMatchSnapshot();
  });
});
