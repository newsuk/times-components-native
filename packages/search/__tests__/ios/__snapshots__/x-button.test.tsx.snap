// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`XButton Should render correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
      "position": "absolute",
      "right": 10,
      "top": 5,
    }
  }
>
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight="24"
    bbWidth="24"
    fill="none"
    focusable={false}
    height="24"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "flex": 0,
          "height": 24,
          "width": 24,
        },
      ]
    }
    vbHeight={24}
    vbWidth={24}
    width="24"
  >
    <RNSVGGroup
      fill={null}
      propList={
        Array [
          "fill",
        ]
      }
    >
      <RNSVGMask
        height="24"
        maskContentUnits={1}
        maskTransform={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        maskUnits={1}
        name="mask0"
        width="24"
        x="0"
        y="0"
      >
        <RNSVGPath
          d="M2 12C2 6.47 6.47 2 12 2C17.53 2 22 6.47 22 12C22 17.53 17.53 22 12 22C6.47 22 2 17.53 2 12ZM15.59 17L17 15.59L13.41 12L17 8.41L15.59 7L12 10.59L8.41 7L7 8.41L10.59 12L7 15.59L8.41 17L12 13.41L15.59 17Z"
          fill={4294967295}
          propList={
            Array [
              "fill",
            ]
          }
        />
      </RNSVGMask>
      <RNSVGGroup
        mask="mask0"
      >
        <RNSVGRect
          fill={4294967295}
          height="24"
          propList={
            Array [
              "fill",
            ]
          }
          width="24"
          x={0}
          y={0}
        />
      </RNSVGGroup>
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
